app/helpers/textual_mixins/textual_availability.rb

```diff
@@ -1,12 +1,13 @@
 module TextualMixins::TextualAvailability
   STATUS_TRANSLATIONS = {
-    'enabled'  => N_('Enabled'),
-    'disabled' => N_('Disabled'),
-    'running'  => N_('Running'),
-    'up'       => N_('Running'),
-    'stopped'  => N_('Stopped'),
-    'down'     => N_('Stopped'),
-    'unknown'  => N_('Unknown')
+    'enabled'   => N_('Enabled'),
+    'disabled'  => N_('Disabled'),
+    'running'   => N_('Running'),
+    'up'        => N_('Running'),
+    'stopped'   => N_('Stopped'),
+    'suspended' => N_('Suspended'),
```

Comment:
I wonder why this is not in place already, since many providers support the suspend state of machines... Do they use different approach? However it seems correct to me.

Reply:
I think so, I was surprised about too.

Reply:
I'm still not convinced this is needed since suspend state works for other providers without this just fine. I don't understand this part of MIQ enough to provide proper review on this.